ComfortZoneApp/app/src/main/java/com/example/comfortzone/listener/DegreeSwitchListener.java

```diff
@@ -0,0 +1,45 @@
+package com.example.comfortzone.listener;
+
+import android.graphics.Typeface;
+import android.view.View;
+import android.widget.TextView;
+
+public class DegreeSwitchListener {
+
+    private TextView tvFahrenheit;
+    private TextView tvCelsius;
+    private Boolean[] isFahrenheit;
+
+    public DegreeSwitchListener(TextView tvFahrenheit, TextView tvCelsius, Boolean[] isFahrenheit) {
```

Comment:
really nice that you delegate this work in single listener class that can be reused!